webpack.blog-utils.config.babel.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable max-len, import/no-extraneous-dependencies */
+import path from 'path';
+
+import webpack from 'webpack';
+import CssMinimizerPlugin from 'css-minimizer-webpack-plugin';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import TerserPlugin from 'terser-webpack-plugin';
+
+import { getPlugins, getRules } from './webpack-common';
+
+const makeConfig = ({ target, externals = {} }) => ({
+  mode: process.env.NODE_ENV,
+  devtool: false,
+  entry: {
+    'index': 'blog-utils',
+  },
+  output: {
+    filename: `${target}.js`,
+    library: {
+      name: 'AddonsFrontendBlogUtils',
+      type: 'umd',
+    },
+    globalObject: 'this',
+  },
+  target,
+  externals,
+  module: {
+    // Set a file limit to embed assets in CSS. It's needed to make this
+    // library easier to use in addons-blog.
+    rules: getRules({ fileLimit: 20000 }),
+  },
+  plugins: [
+    ...getPlugins({ withBrowserWindow: target === 'web' }),
+    new webpack.NormalModuleReplacementPlugin(
+      /amo\/tracking/,
+      'blog-utils/tracking.js',
```

Comment:
I gather this is temporary until we support GA for the blog?

Reply:
Not necessarily. This library is going to be used at build time only. The "runtime" JS will likely live in the addons-blog repo, or it will be a different lib and/or bundle.